didact/camelk/first-integration.md

```diff
@@ -0,0 +1,132 @@
+
+Users of Camel K can instantly run integration code written in Camel DSL on their preferred cloud (Kubernetes or OpenShift).
+
+[Check out the Apache Camel K project documentation for more details about the framework.](https://camel.apache.org/camel-k/latest/index.html)
+
+## Apache Camel K in VS Code - Your First Integration
+
+What follows is a simple step-by-step process that helps you create and deploy a Camel K integration on a Minikube instance running locally.
+
+We will:
+
+* Create a folder with a sample Apache Camel integration written in Groovy
+* Deploy the file in a local Minikube instance
+* Update the Groovy file and watch the change ripple through to the running integration in seconds! 
+
+## Prerequisites 
+
+You must have a few things set up prior to walking through the steps in this tutorial. 
+
+<a href='didact://?commandId=vscode.didact.validateAllRequirements' title='Validate all requirements!'><button>Validate all Requirements at Once!</button></a>
+
+| Requirement (Click to Verify)  | Availability | Additional Information/Solution |
+| :--- | :--- | :--- |
+| [At least one folder exists in the workspace](didact://?commandId=vscode.didact.workspaceFolderExistsCheck&text=workspace-folder-status "Ensure that at least one folder exists in the user workspace"){.didact} | *Status: unknown*{#workspace-folder-status} | Create a workspace folder (or [click here to create a temporary folder](didact://?commandId=vscode.didact.createWorkspaceFolder "Create a temporary folder and add it to the workspace."){.didact}), close, and reopen the Didact window
+| [Is Kamel available?](didact://?commandId=vscode.didact.cliCommandSuccessful&text=kamel-status$$kamel "Tests to see if `kamel` returns a result"){.didact} 	| *Status: unknown*{#kamel-status} 	| See [Installing Camel K](https://camel.apache.org/camel-k/latest/installation/installation.html "Documentation on how to Install Apache Camel K")
+| [VS Code Extension Pack for Apache Camel by Red Hat is installed](didact://?commandId=vscode.didact.extensionRequirementCheck&text=extension-requirement-status$$redhat.apache-camel-extension-pack "Checks the VS Code workspace to make sure the extension pack is installed"){.didact} | *Status: unknown*{#extension-requirement-status} 	| [Click here to install](vscode:extension/redhat.apache-camel-extension-pack "Opens the extension page and provides an install link") |
+| [VS Code Tooling for Apache Camel K by Red Hat is installed](didact://?commandId=vscode.didact.extensionRequirementCheck&text=camelk-extension-requirement-status$$redhat.vscode-camelk "Checks the VS Code workspace to make sure the extension pack is installed"){.didact} | *Status: unknown*{#camelk-extension-requirement-status} 	| [Click here to install](vscode:extension/redhat.vscode-camelk "Opens the extension page and provides an install link") |
```

Comment:
the VS Code Apache Camel K check can be removed it is coming with the Extension pack

Reply:
As of today it is not listed as part of the extension pack - https://marketplace.visualstudio.com/items?itemName=redhat.apache-camel-extension-pack

Reply:
it is listed even though it is missing a specifc dot.

![Screenshot from 2020-06-03 16-16-46](https://user-images.githubusercontent.com/1105127/83648072-e9e53e00-a5b5-11ea-95f5-ad5155c4740a.png)

we can check that it is available here https://github.com/camel-tooling/vscode-camel-extension-pack/blob/350b0e126e3f0d1e75b50c6af6079937a29fbfe3/package.json#L28